tools/doc/allhtml.js

```diff
@@ -73,3 +73,13 @@ all = all.slice(0, apiStart.index + apiStart[0].length) +
 
 // Write results.
 fs.writeFileSync(source + '/all.html', all, 'utf8');
+
+// Validate all hrefs have a target
+const ids = {};
```

Comment:
Can we use a `Set` for this? I think that might be a tad more natural :slightly_smiling_face: